src/mono/wasm/runtime/marshal-to-js.ts

```diff
@@ -483,7 +483,7 @@ function _marshal_span_to_js(arg: JSMarshalerArgument, element_type?: MarshalerT
         result = new Span(<any>buffer_ptr, length, MemoryViewType.Double);
     }
     else {
-        throw new Error(`NotImplementedException ${MarshalerType[element_type]} `);
+        throw new Error(`NotImplementedException ${MarshalerType[element_type]}. For more information see https://aka.ms/dotnet-wasm-jsinterop`);
```

Comment:
perhaps we could have `const  jsinteropDoc="For more information see https://aka.ms/dotnet-wasm-jsinterop"`